### ISSUE TITLE
What if "apt-get" becomes an instruction?

### ISSUE THREAD

**Issue by limjcst** [containerbuildsystem/dockerfile-parse#91]
Inside a multi-line RUN instruction, lack of `\` will lead missing of command and no exception will be raised.

Here is an example Dockerfile
```
FROM ubuntu:xenial

RUN apt-get install -y curl && \
    curl --silent --location https://deb.nodesource.com/setup_4.x | bash - && \
    apt-get install -y nodejs
    apt-get install -y build-essentials
    echo Hello
```

Now, `echo` is taken as an instruction, but `apt-get` is lost.

If `apt-get` can also be taken as an instruction, I can deal with this Dockerfile syntax error manually.
If not, an exception is expected.

**Comment by MartinBasti** [comment]
Hello,
your Dockerfile is invalid. You have to specify `RUN` before each commad

```
FROM ubuntu:xenial

RUN apt-get install -y curl && \
    curl --silent --location https://deb.nodesource.com/setup_4.x | bash - && \
    apt-get install -y nodejs
RUN apt-get install -y build-essentials
RUN echo Hello
```

**Comment by limjcst** [comment]
Thanks for your attention!

You are right.
And the Dockerfile is supposed to be
```
FROM ubuntu:xenial

RUN apt-get install -y curl && \
    curl --silent --location https://deb.nodesource.com/setup_4.x | bash - && \
    apt-get install -y nodejs && \
    apt-get install -y build-essentials && \
    echo Hello
```

I think it would be better if `dockerfile-parse` could handle invalid input.

**Comment by limjcst** [comment]
This issue may be related to https://github.com/containerbuildsystem/dockerfile-parse/blob/47740325fb821af04c4c94cd4d59701cbcd3f194/dockerfile_parse/parser.py#L243

as `apt-get` does not match `^\s*(\w+)\s`.

How about replace the regex with `r'^\s*(\S+)\s+(.*)$'`?
In this way, Dockerfile error is left for the user to deal with.

**Comment by MartinBasti** [comment]
Yeah it indeed seems like a bug in dockerfile-parse. Would you mind to contribute with a PR (including unit-test)?

**Comment by limjcst** [comment]
My pleasure.

BTW, the original code cannot pass flake8 :(

**Comment by MartinBasti** [comment]
That's possible, at least all new code must pass flake8 testing, we have enabled stickler-CI to cover new things.

Fixed via #92
